products/global-cli-configs/cli-api/grid-cluster-context.md

```diff
@@ -0,0 +1,32 @@
+---
+description: Sets the cluster context for the user.
+---
+
+# grid user set-cluster-context
+
+With multi-cluster availability, if you are working against a particular cluster, it's easier to set the
+cluster context so all your CLI actions (including creation of a resource such as Run or Session) are going
+against that cluster. Apart from the easiness this command brings, it also help you avoid mistakes such
+as scheduling to a wrong cluster. 
+
```

Comment:
*Please Note: This feature will evolve to include more elaborate cluster management in the near-term future.